app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -263,7 +263,7 @@ abstract class BaseBrowserFragment : Fragment(), BackHandler, SessionManager.Obs
                 }
             )
 
-            downloadFeature.onDownloadCompleted = { download, _, downloadJobStatus ->
+            downloadFeature.onDownloadStopped = { download, _, downloadJobStatus ->
```

Comment:
I had to make that change because this was renamed in `android-components` and the version wasn't increased, if you resync your project it won't build anymore.